integrations/integration.php

```diff
@@ -48,15 +49,22 @@ abstract class Integration {
 	protected bool $is_active = false;
 
 	/**
-	 * Instance of VipIntegrationConfig. It's useful to have full configuration info
+	 * Array containing all configuration data. It's useful to have full configuration info
 	 * available inside each integration, we can use it for cases like multisite,
 	 * tracking etc.
 	 *
 	 * Note: We don't use this property for activation of the integration.
 	 *
-	 * @var IntegrationVipConfig
+	 * @var array
 	 */
-	private IntegrationVipConfig $vip_config;
+	private array $vip_configs = [];
+
+	/**
+	 * A boolean indicating if the integration have multiple configs.
+	 *
+	 * @var bool
+	 */
+	protected bool $have_multiple_configs = false;
```

Comment:
For backwards-compat, should we default this to `true`? It looks like a lot of the integrations seem to have more than one config.

Reply:
Seems like we should change the variable name to `$have_multiple_instances` because here I am not referring about the config keys of the integration but multiple configurations of the same integration.

Does the new name make sense? feel free to let me know if you have a better suggestion :)

Reply:
Ahhhh, then yes. Let's change the name because that definitely confused me! Maybe even `$has_multiple_setups`?

Reply:
Fixed [251976d](https://github.com/Automattic/vip-go-mu-plugins/pull/5643/commits/251976d22dd5e4774453f0197b053eea5c0e8fa6)